src/wp-includes/class-wp-html-span.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * HTML Span: Represents a textual span inside an HTML document.
+ *
+ * @package WordPress
+ * @subpackage HTML
+ * @since 6.2.0
+ */
+
+if ( ! class_exists( 'WP_HTML_Span' ) ) :
```

Comment:
Remove this guard. See https://github.com/WordPress/wordpress-develop/pull/3920/files#r1090712903 for reasoning.